testing/integration/install_test.go

```diff
@@ -250,6 +258,50 @@ func TestInstallPrivilegedWithBasePath(t *testing.T) {
 	t.Run("check second agent installs with --develop", testSecondAgentCanInstall(ctx, fixture, randomBasePath, true, opts))
 }
 
+func testInstallWithoutBasePathWithCustomUser(ctx context.Context, t *testing.T, fixture *atesting.Fixture, customUsername, customGroup string) {
+	// Run `elastic-agent install`.  We use `--force` to prevent interactive
+	// execution.
+	// create testing user
+	if customUsername != "" {
+		pt := progressbar.NewOptions(-1)
+		_, err := install.EnsureUserAndGroup(customUsername, customGroup, pt, true)
+		require.NoError(t, err)
+	}
+
+	opts := atesting.InstallOpts{Force: true, Privileged: false, Username: customUsername, Group: customGroup}
+	out, err := fixture.Install(ctx, &opts)
+	if err != nil {
+		t.Logf("install output: %s", out)
+		require.NoError(t, err)
+	}
+
+	// Check that Agent was installed in default base path
+	topPath := installtest.DefaultTopPath()
+	checks := &installtest.CheckOpts{
+		Privileged: opts.Privileged,
+		Username:   customUsername,
+		Group:      customGroup,
+	}
+	require.NoError(t, installtest.CheckSuccess(ctx, fixture, topPath, checks))
```

Comment:
It looks to me like the test for Windows is slightly weaker than the test for Linux here, because Windows does not try to execute the Elastic Agent status command as the relevant user, because that requires more than just a single command on Windows. 

Is there a way for us to do this? The test right now has us using a privileged user to create a less privileged user, but doesn't seem to try to interact with the agent as that less privileged user. 

* Windows just checks the permissions SIDs https://github.com/elastic/elastic-agent/blob/434986ed9fa6fbed6257557a4c6773d519742f66/testing/installtest/checks_windows.go#L55-L60
* Linux checks the permissions, UID+GID, and makes sure the commands work from the perspective of the relevant user https://github.com/elastic/elastic-agent/blob/434986ed9fa6fbed6257557a4c6773d519742f66/testing/installtest/checks_unix.go#L25-L34

Reply:
> Is there a way for us to do this? The test right now has us using a privileged user to create a less privileged user, but doesn't seem to try to interact with the agent as that less privileged user.

It is not possible to execute as the `elastic-agent-user` because the user is created without the ability to login or get a real execution shell. The only way to actually test this would be to create another user, add that user to the `elastic-agent` group and then run the command as that user.